### PR TITLE
add-logs-for-label-operations

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 1]
```diff
@@ -3257,16 +3256,19 @@ public JobLabelInfo updateLabel(String labelId, String newLabel)
...
 
     public void deleteLabel(String labelId) throws LabelNotFoundException {
         if (!checkIfLabelIdExists(Long.parseLong(labelId))) {
             throw new LabelNotFoundException(labelId);
         }
+        String label = getLabelById(Long.parseLong(labelId));
         executeReadWriteTransaction(session -> session.getNamedQuery("deleteLabel")
                                                       .setParameter("labelId", Long.parseLong(labelId))
                                                       .executeUpdate());
+        logger.info("Label " + label + " set been deleted");
```
Comment (aminelouati) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3271]: "Label " + label + " set been deleted" --> Label " + label + " has been deleted"

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 2]
```diff
@@ -3300,6 +3303,7 @@ public void removeJobLabels(List<String> jobIds) {
                                                                                                .setParameter("jobIdList",
                                                                                                              longIds)
                                                                                                .executeUpdate()));
+            logger.info("Label set been removed from jobs " + jobIds);
```
Comment (aminelouati) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3306]: this logger message is not clear

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 3]
```diff
@@ -3192,6 +3189,7 @@ public List<JobLabelInfo> newLabels(List<String> labels) throws LabelConflictExc
             JobLabel jobLabel = executeReadWriteTransaction(session -> {
                 JobLabel labelData = JobLabel.createJobLabel(label);
                 session.save(labelData);
+                logger.info("Label " + label + " has been created");
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3192]: it is better to put this log message inside the frontend caller method.
This way, you can include in the log message the user which created the label

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 4]
```diff
@@ -3211,6 +3209,7 @@ public List<JobLabelInfo> setLabels(List<String> labels) throws LabelValidationE
             JobLabel jobLabel = executeReadWriteTransaction(session -> {
                 JobLabel labelData = JobLabel.createJobLabel(label);
                 session.save(labelData);
+                logger.info("Label " + label + " set been set");
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3212]: same remark as above

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 5]
```diff
@@ -3257,16 +3256,19 @@ public JobLabelInfo updateLabel(String labelId, String newLabel)
                                                       .setParameter("newLabel", newLabel)
                                                       .setParameter("labelId", Long.parseLong(labelId))
                                                       .executeUpdate());
+        logger.info("Label " + newLabel + " set been updated");
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3259]: same

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java (hunk 1 above)
Comment (fviale) [reply scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3271]: same remark as above

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 6]
```diff
@@ -3284,6 +3286,7 @@ public void setLabelOnJobIds(String labelId, List<String> jobIds) throws LabelNo
                                                                                                .setParameter("jobIdList",
                                                                                                              jobIdSubList)
                                                                                                .executeUpdate()));
+            logger.info("Label " + label + " set been set of jobs " + jobIds);
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3289]: same remark as above

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java (hunk 2 above)
Comment (fviale) [reply scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3306]: same remark as above

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java (hunk 3 above)
Comment (oanaBiancaSchiopu) [reply scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3192]: For some operation we don't have the label name in the frontend caller method. We receive the Id as request param

Comment (fviale) [reply scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3192]: alright, in that case, you could add the intiator as a parameter to the database methods. This way you would have both information when printing the logs.

**Review by fviale — APPROVED** [review]
LGTM

**Pull request merged**